test/rage2e/rag_test.go

```diff
@@ -67,21 +71,92 @@ var _ = Describe("RAGEngine", func() {
 		}
 	})
 
+	It("should create RAG with localembedding and kaito VLLM workspace successfully", func() {
+		numOfReplica := 1
+		workspaceObj := createPhi3WorkspaceWithPresetPublicModeAndVLLM(numOfReplica)
+
+		time.Sleep(30 * time.Second)
+
+		validateCreateNode(workspaceObj, numOfReplica)
```

Comment:
"validateCreateNode" do we need this step? If node has problem, the next validateWorkspaceResourceStatus should fail.